VKClone/VKClone/Model/Entity/Group.swift

```diff
@@ -0,0 +1,8 @@
+// Group.swift
+// Copyright © Roadmap. All rights reserved.
+
+import Foundation
+struct Group {
```

Comment:
```suggestion
struct Group {
```
линтер если работает то должен заставить комент написать если не работает осуждаю!

Reply:
Работает, не заставляет потому что выше строчка есть)